### DISCUSSION TITLE
[BETA] Release version v0.2.1 evolved to 0.2.1.0007!

### DISCUSSION THREAD

**Discussion by zhenrong-wang** [zhenrong-wang/hpc-now#12] [Announcements]
This is a minor evolution of Version 0.2.1.0006. 

Main changes:

1. Validated and enhanced the batch mode. 
2. Fixed some known bugs.
3. Adjusted the interactive prompts.

Please fork, build, test, and run. Any issues found, please submit to this repo.

*HPC-NOW: Start your HPC journey in the cloud now.*

<hr /><em>This discussion was created from the release <a href='https://github.com/zhenrong-wang/hpc-now/releases/tag/0.2.1.beta.7'>[BETA] Release version v0.2.1 evolved to 0.2.1.0007!</a>.</em>

**Comment by caozying** [comment]
Testing the batch mode now. Successfully created and destroyed clusters in Alibaba Cloud.

**Reply by zhenrong-wang** [reply]
OK, I am thinking about move the macros from now_macros.h to a global configuration file and let the hpcopr read the configurations when start. 

Then, users can set batch mode as their default mode.

A good idea?
